about.html

```diff
@@ -5,10 +5,120 @@
+                    has not been addressed across the country through previous conferences and activities, and this is
+                    what our conference
+                    aims to achieve.
+                    What can we look forward to in #IT_Live_2017?
+                    There will be a number of experts in this field discussing different topics, a number of
+                    competitions, entertainment
+                    events and gifts, companies and individuals with experience in this field who will present their
+                    ideas and other
+                    activities that will be announced soon.
+                    When is the conference date?
+                    End of February 2017, dates and times will be announced soon along with the application form for
+                    attendees and speakers.</p>
+                <div class="about__contact flex column">
+                    <p class="about--peragraph">
+                        Please contact us per Email for any further questions about CC Global Summit 2015!
+                    </p>
+                    <a href="#">info@itlive.iq</a>
+                </div>
+            </div>
+            <div class="about__logo flex column">
+                <h2 class="section--title">IT.LIVE IRAQ TECH SUMMIT 2019 Logo</h2>
+                <p class="about--peragraph">The logo of IT.LIVE IRAQ TECH SUMMIT 2019 was decided through the logo
+                    competition
+                    from 8. June to 7. July.</p>
+                <img src="images/brands/it_live.png" alt="" class="globale__summit__logo">
+            </div>
+            <div class="past__summit flex column">
+                <h2 class="section--title">See the past IT.LIVE Summits</h2>
+                <p class="about--peragraph">Take a look at the last two IT.LIVE Summits which took place in Baghdad.</p>
+                <div class="summit__group flex column">
+                    <div class="summit">
+                        <img src="images/it_live_2018.jpeg" alt="it live 2018 Iraq, Baghdad">
+                        <span class="image__overlay"></span>
+                        <h3 class="summit__date">2018</h3>
+                        <p class="summit__location">Iraq, Baghdad</p>
+                    </div>
+                    <div class="summit">
+                        <img src="images/it_live_2017.jpeg" alt="it live 2017 Iraq, Baghdad">
+                        <span class="image__overlay"></span>
+                        <h3 class="summit__date">2017</h3>
+                        <p class="summit__location">Iraq, Baghdad</p>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="partners flex column bg-dark" id="partners">
+            <h2 class="section--title">Partners</h2>
+            <ul class="nav--brands">
+                <li><img src="images/brands/careem.svg" alt="mozilla co. brand logo"></li>
```

Comment:
- [ ] An image is broken on your design, It would be better to fix it so your design is professional. try to change `careem.svg` to `Careem.svg`.

<img width="947" alt="image" src="https://user-images.githubusercontent.com/77190203/158038395-528de812-f808-4bab-88de-12c9f098dfde.png">